plugins/database/mongodb/certs_test.go

```diff
@@ -0,0 +1,129 @@
+package mongodb
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"testing"
+	"time"
+)
+
+type Cert struct {
+	Key    *rsa.PrivateKey
+	KeyPem []byte
+
+	Cert    *x509.Certificate
+	CertPem []byte
+
+	TLSCert tls.Certificate
+}
+
+func MakeCert(t *testing.T, parent *x509.Certificate) (bundle Cert) {
```

Comment:
* Why export this?
* Is this also a `t.Helper()`?

Reply:
Refactored as a part of the integration test